sovrin_client/test/cli/test_connect_env.py

```diff
@@ -5,24 +5,44 @@
 
 
 @pytest.fixture(scope="module")
-def aliceCli(aliceCLI):
+def alice(aliceCLI):
```

Comment:
Why be less explicit, what is `alice`, CLI, wallet, agent? Once you have a few more players, it start getting confusing.